Project/src/main/java/app/models/Article.java

```diff
@@ -0,0 +1,141 @@
+package app.models;
```

Comment:
I imagine this is the refactored file class. I mentioned it at the start of the PR that I will tell you what could have been its own PR.

Refactoring an integral part of the code is important and should receive its own PR.

Reply:
I started working on the Article class before the File class was created. This wasn't a refactor, please check my branch history. I know that it may seem like I did a lot of refactoring but that is because I had to integrate what I had already written (ArticleStatus, Article, etc.) for my issues as people submitted their PR's before me.

I hear what you're saying, but I wanted to clarify that I didn't just go out cowboy and change/refactor a bunch of stuff unrelated to my issues. I did rename a bunch of things to match what I had already written, and that was probably not the right choice but I couldn't just drop all of my previous work.